assembl/static2/tests/unit/reducers/adminReducer/voteSession.spec.js

```diff
@@ -1155,5 +1274,264 @@ describe('voteSession admin reducers', () => {
       const actual = modulesById(state, action);
       expect(actual.toJS()).toEqual(expected);
     });
+
+    it('should handle CANCEL_MODULE_CUSTOMIZATION action type (choice gauge)', () => {
+      const state = fromJS({
+        customGauge: {
+          _hasChanged: false,
+          _isNew: false,
+          _toDelete: false,
+          isCustom: true,
+          isNumberGauge: false,
+          id: 'customGauge',
+          voteSpecTemplateId: 'template',
+          proposalId: 'proposal1',
+          instructionsEntries: [
+            {
+              localeCode: 'en',
+              value: 'My custom instructions'
+            }
+          ],
+          choices: ['custom-choice1', 'custom-choice2']
+        },
+        template: {
+          _hasChanged: false,
+          _isNew: false,
+          _toDelete: true,
```

Comment:
`_toDelete: true` is this normal?

Reply:
yes, whatever. We just have to ensure that the template data remains unchanged.